week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,48 +67,59 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+
+
+function formatPercentage(arr) {
+    let newArr = [];
+    for (i = 0; i < arr.length; i++) {
+        if (arr[i] > 100) {
+            arr[i] = 100;
+        }
+        newArr[i] = arr[i].toFixed(2);
```

Comment:
There is small issue here, try to run the test and check. due to the nature of numbers being float